pinot-common/src/main/java/org/apache/pinot/common/utils/SimpleHttpErrorInfo.java

```diff
@@ -30,6 +31,7 @@ public class SimpleHttpErrorInfo {
   private String _error;
 
   @JsonCreator
+  @JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
I added this because I saw an issue in our prod cluster where the controller was not logging the exact error message returned by the server because an unexpected field was returned in the response (I think it was `status`).